planter/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This scrpt is the entypoint for the planter docker image
+set -x
+# write a fake user entry with settings matching the host user possible
+echo "$USER:!:$UID:$GID:$HOME:/bin/bash" > /etc/passwd
+cd ${DIR}
```

Comment:
you can probably skip this by passing `-w "${PWD}"` to the `docker run` command.

Reply:
done.